lib/stat_tracker.rb

```diff
@@ -184,6 +184,50 @@ def worst_coach(for_season)
      end
 
     game_team_by_coach.key(game_team_by_coach.values.min)
-    require "pry"; binding.pry
+  end
+
```

Comment:
These are all clean, but I think if there is time we can definitely refactor into it being a helper method in one of the other files since they are all pretty much doing the same thing.